src/wp-includes/class-wp-customize-manager.php

```diff
@@ -2106,6 +2109,7 @@ public function remove_frameless_preview_messenger_channel() {
 		} )();
 		</script>
 		<?php
+		wp_print_inline_script_tag( str_replace( array( '<script>', '</script>' ), '', ob_get_clean() ) );
```

Comment:
oops, we don't want to arbitrarily replace these tags. we _know_ they exist at the front and back of the string, so by manually removing them we can avoid unintentional HTML syntax poisoning.

```php
$script_html = ob_get_clean();
$script_html = substr( $script_html, strlen( '<script>' ), -strlen( '</script>' ) );
```

the `strlen` calls shouldn't add any overhead because that's stored in the string object, which here is a string literal, which we've already created in this patch inside the array.

this change is both safer, deterministic, and more efficient, particularly in the worse-case, though given that we expect very short args here I would be surprised if it's a measurable impact. still, doing less is faster than doing more.

Reply:
Great catch @dmsnell!

@westonruter IMO this is worth a quick follow up commit applying this throughout.

If we feel strongly about introducing a helper at this point, I wouldn't be opposed to that either. Though I think that should remain an `@access private` function if we add it.

Reply:
Private helper function added in https://github.com/WordPress/wordpress-develop/pull/5301